app/styles/ui/_preferences.scss

```diff
@@ -83,7 +83,7 @@
   .theme-selector {
     display: flex;
     flex-wrap: wrap;
-    margin: 0 calc(var(--spacing-half) * -1);
+    margin: 0 calc(var(--spacing-half) * -1) var(--spacing-double);
```

Comment:
:art: what's the context on this addition?

Reply:
That is for the margin between Theme and the newly added "Title bar style" section. But I found it better be handled with `<Row></Row>` instead. Dropping.